xmipp

```diff
@@ -449,7 +449,8 @@ def printVersion():
     with open(VERSION_FILE, 'w') as f:
         f.write(str(log))
         f.write('\n  nvcc version:     ')
-        f.write(log3)
+        if log3 != []:
```

Comment:
More elegant:
```Python
if log3:
    f.write(log3)
```
That's because if log3 is an empty list, if log3 will be false, and if it's not empty, true.